sentences/sv/binary_sensor_HassGetState.yaml

```diff
@@ -0,0 +1,278 @@
+language: "sv"
+intents:
+  HassGetState:
+    data:
+      # https://www.home-assistant.io/integrations/binary_sensor/
+      # Battery
+      - sentences:
+          - "är <name> [batteri[et]] {bs_battery_states:state} [<i_på> <area>q]"
+          - "är batteriet [till] <name> {bs_battery_states:state} [<i_på> <area>q]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: battery
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      - sentences:
+          - "är (något|några) batteri[er] {bs_battery_states:state} [<i_på> <area>q]"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      - sentences:
+          - "är alla batteri[er] {bs_battery_states:state} [<i_på> <area>q]"
+        response: all
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      - sentences:
+          - "<vilka> (batteri|enhet)[er] är {bs_battery_states:state} [<i_på> <area>q]"
+        response: which
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      - sentences:
+          - "hur många (batteri|enhet)[er] är {bs_battery_states:state} [<i_på> <area>q]"
+        response: how_many
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      # Battery charging
+      - sentences:
+          - "är <name> [batteri|[er]] {bs_battery_charging_states:state} [<i_på> <area>q]"
+          - "{bs_battery_charging_states:state} <name> [batteri[er]] [<i_på> <area>q]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: battery_charging
+        slots:
+          domain: binary_sensor
+          device_class: battery_charging
+
+      - sentences:
+          - "är [det] (något|något) (batteri|enhet)[er] {bs_battery_charging_states:state} [<i_på> <area>q]"
```

Comment:
might need "som" in this sentence, maybe add "som" as skip_word or as an optional to this sentence
eg:
"är det någon enhet som laddar i köket"